src/libPMacc/include/cuSTL/cursor/tools/LinearInterp.hpp

```diff
@@ -61,13 +66,13 @@ struct LinearInterp1D
 namespace result_of
 {
 
-template<typename T_Position, typename TCursor>
-struct Functor<cursor::tools::LinearInterp1D<T_Position>, TCursor>
+template<typename T_Cursor, typename T_PositionComp>
+struct Functor<cursor::tools::LinearInterp<T_PositionComp>, T_Cursor>
 {
-    typedef cursor::Cursor<
-        cursor::LinearInterpAccessor1D<TCursor, T_Position>,
-        cursor::PlusNavigator,
-        T_Position> type;
+    typedef PMacc::cursor::Cursor<cursor::LinearInterpAccessor<T_Cursor>,
```

Comment:
Maybe use the new indention style to avoid long lines

```C++
typedef PMacc::cursor::Cursor<
    cursor::LinearInterpAccessor<T_Cursor>,
    cursor::PlusNavigator,
    PMacc::math::Vector<
        T_PositionComp,
        PMacc::cursor::traits::dim<
            T_Cursor
        >::value
    >
> type;
```